src/main/frontend/handler/file_sync.cljs

```diff
@@ -14,9 +14,13 @@
 
 (def *beta-unavailable? (volatile! false))
 
-(def hiding-login&file-sync (not config/enable-file-sync?))
 (def refresh-file-sync-component (atom false))
 
+(defn enable-sync?
+  []
+  (or (state/enable-sync?)
+      config/dev?))
```

Comment:
```
(defonce dev? ^boolean (or dev-release? goog.DEBUG))
```

```
"cljs:release-electron": "clojure -M:cljs release app electron --debug && clojure -M:cljs release publishing",
```

So this flag is wrong.

Reply:
I don't think `shadow-cljs release app --debug` means `goog.DEBUG`, it's still advanced mode but have some prettier output.

Reply:
Source: https://github.com/thheller/shadow-cljs/blob/9a4144b65141ecdacc28fefed53fe968035363e5/src/main/shadow/cljs/devtools/cli_opts.cljc#L78